CMakeLists.txt

```diff
@@ -11,23 +11,8 @@ file(GLOB_RECURSE GLSL "resources/*.glsl" "src/*.glsl")
 # Set the executable.
```

Comment:
This is CMake file is specific to your build environment. Can you uncommit it and leave it uncommitted for future PRs?